frontend/src/component/common/ConstraintAccordion/ConstraintAccordionView/ConstraintAccordionViewHeader/ConstraintAccordionViewHeader.tsx

```diff
@@ -34,6 +35,13 @@ export const ConstraintAccordionViewHeader = ({
     expanded,
     compact,
 }: IConstraintAccordionViewHeaderProps) => {
+    const { context } = useUnleashContext();
+    const { contextName } = constraint;
+
+    const disableEdit = !context
+        .map(contextDefinition => contextDefinition.name)
```

Comment:
```suggestion
        .map(({name}) => name)
```
Another concise way to do